core/src/main/java/org/elasticsearch/action/admin/indices/template/put/PutIndexTemplateRequest.java

```diff
@@ -473,7 +471,7 @@ public void readFrom(StreamInput in) throws IOException {
         cause = in.readString();
         name = in.readString();
 
-        if (in.getVersion().onOrAfter(V_5_1_0)) {
+        if (in.getVersion().onOrAfter(Version.V_6_0_0_alpha1)) {
```

Comment:
Good catch. This was my mistake. I thought until the last minute that that change was going back to 5.1 but we decided against and I didn't fix this.